.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+<!--Hi there! If you aren't submitting a blog post, you can delete this message.
+
+Thank you for submitting a blog post! We want to make sure content produced for Quansight Labs is accessible, so please address the the following guidelines in your post.-->
+
+
+Text styling
+- [ ] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
+- [ ] If there are headers, they use the proper header tags in order to do so.
+- [ ] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.
+
+Non-text content
+- [ ] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
```

Comment:
what about links? - I have zero idea how to phrase it though

- **bad**: check [here](https://labs.quansight.org/)
- **good**: check the [Quansight labs website](https://labs.quansight.org/)